platform/ethereum/model.go

```diff
@@ -61,7 +63,7 @@ type Collection struct {
 	ImageUrl    string                 `json:"image_url"`
 	Description string                 `json:"description"`
 	ExternalUrl string                 `json:"external_url"`
-	Total       int                    `json:"owned_asset_count"`
+	Total       *big.Int               `json:"owned_asset_count"`
```

Comment:
Can we keep this as int?

Reply:
I got one case with the `owned_asset_count` equal to `20000000000000000000`, and we got an Unmarshal error